misc/bazel/rust.bzl

```diff
@@ -8,7 +8,7 @@ def codeql_rust_binary(
         visibility = None,
         symbols_test = True,
         **kwargs):
-    rust_label_name = name + "_single_arch"
+    rust_label_name = "single_arch/" + name
```

Comment:
❤️

Reply:
(mind that this is not just stylistic, by stubbing this with an `alias` this was required to preserve the base name)